packages/amplify_api/ios/Classes/ApiBridge.swift

```diff
@@ -35,18 +35,28 @@ class ApiBridge {
         return Amplify.API.delete(request: request, listener: listener)
     }
     
-    
-    func query<R: Decodable>(request: GraphQLRequest<R>,
+    func head(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
+        return Amplify.API.head(request: request, listener: listener)
+    }
+
+    func patch(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
+        return Amplify.API.patch(request: request, listener: listener)
+    }
+
+
+    @discardableResult
+    public func query<R: Decodable>(request: GraphQLRequest<R>,
                                     listener: GraphQLOperation<R>.ResultListener?) -> GraphQLOperation<R> {
         return Amplify.API.query(request: request, listener: listener)
     }
 
-    func mutate<R: Decodable>(request: GraphQLRequest<R>,
+    @discardableResult
```

Comment:
Do we need the `@discardableResult` annotation?

Reply:
I guess not as we are actually using the result each time due to our cancel support code.